mycroft/skills/main.py

```diff
@@ -31,6 +33,20 @@
 def load_skills_callback():
     global client
     load_skills(client)
+    config = ConfigurationManager.get()
+    config_core = config.get("core")
+
+    try:
```

Comment:
So, i guess the intention of having the ini third party skills dir /and/ the /opt/mycroft 3p skills dir is so that we can have skills in more than one location?